drivers/CoreVideo/include/JPEGImageProperties.hpp

```diff
@@ -0,0 +1,17 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <cstdint>
+
+#include "interface/drivers/STM32Hal.h"
+
+namespace leka {
+
+struct JPEGImageProperties : JPEG_ConfTypeDef {
+	[[nodiscard]] auto getWidthOffset() const -> uint32_t;
+};
```

Comment:
ça c'est chic de faire une composition de struct à partir de celle de ST pour avoir la notre.

c'est un trick à garder en tête pour toutes les autres interfaces, même dans le BLE par exemple.

je rajoute juste `public` pour expliciter l'inheritance.
 
```suggestion
struct JPEGImageProperties : public JPEG_ConfTypeDef {
	[[nodiscard]] auto getWidthOffset() const -> uint32_t;
};
```